oak_crypto/BUILD

```diff
@@ -33,7 +33,7 @@ rust_library(
         "@oak_crates_index//:async-trait",
     ],
     deps = [
-        "//proto/crypto:crypto_rust_prost",
+        "//oak_proto_rust",
```

Comment:
Does it still build with the crypto_rust_prost target?

Reply:
Not since I moved the conditional compilation logic from oak_crypto/src/lib.rs to oak_proto_rust/src/lib.rs.

Reply:
Note for follow up PR: as proposed by @jblebrun , we can normalise the import path in .rs files having something like

```
#[cfg(not(feature = "bazel"))]
pub mod crypto_proto {
    pub mod crypto {
        pub use oak_proto_rust::oak::crypto::*;
    }
}
```

in oak_proto_rust/src/lib.rs . This will make it easier to transition out of `oak_proto_rust` crate in the future.

Reply:
As discussed, we can't yet build crypto.proto with "//proto/crypto:crypto_rust_prost", because we need it built with micro_rpc.